### ISSUE TITLE
PowerShellOnTargetMachinesV3: Make 30 second delay while checking job status configurable

### ISSUE THREAD

**Issue by wgrthoen** [microsoft/azure-pipelines-tasks#10641]
## Required Information

**Question, Bug, or Feature?**  
*Type*: Feature

**Enter Task Name**: PowerShellOnTargetMachinesV3

## Environment
SF / releaseId=17078 & environmentId=83331
Private agent

## Issue Description

Executing remote PowerShell on target machine V3 takes 33 seconds while the powershell script is normally finished in 3-5 seconds. Because the task depends on https://github.com/microsoft/azure-pipelines-extensions/blob/master/TaskModules/powershell/RemoteDeployer

See why here: https://github.com/microsoft/azure-pipelines-extensions/issues/632#issue-455174357

I'd rather have the Start-Sleep value configured, in ms, from this task which would require work on both repositories.

### Task logs
```
2019-06-12T12:11:01.3753067Z VERBOSE: JobId: '2', JobState: 'running', ComputerName: 'otd0937-t.xxx.tld'
2019-06-12T12:11:01.3762516Z ================================================ otd0937-t.xxx.tld ================================================
2019-06-12T12:11:31.4202557Z VERBOSE: JobId: '2', JobState: 'completed', ComputerName: 'otd0937-t.xxx.tld'
2019-06-12T12:11:31.4203720Z ================================================ otd0937-t.xxx.tld ================================================
```
## Troubleshooting

None

### Error logs

None

**Comment by arjgupta** [comment]
closing this issue. Relevant PR: https://github.com/microsoft/azure-pipelines-tasks/pull/10661

**Comment by wgrthoen** [comment]
Why is this issue closed?

I can see th PR lowering the start-sleep to 5 seconds. This issue request for making it configurable. Is that not happening with a specific reason?

**Comment by arjgupta** [comment]
@wgrthoen we do not think providing an extra input will be the right call here. Instead we are reducing the sleep time.

**Comment by centreboard** [comment]
@argupta Is this available with Azure DevOps Server 2019.1?
Having updated we're getting version 3.1.2 of this task currently, while the updated version is 3.1.3?
